lib/thinking_sphinx/masks/pagination_mask.rb

```diff
@@ -51,7 +51,7 @@ def total_entries
   def total_pages
     return 0 if search.meta['total'].nil?
```

Comment:
It makes more sense to check here for `total_found` instead of `total` because later on we don't use this property.

But either way, in manticoresoftware/manticoresearch#912 manticore team asks for a reproducer. Any idea how to get these properties `total`, `total_found`, etc. with plain SQL commands?

I'm hitting this issue with wrong number of pages with Manticore 6.2.12 in simple unit tests. And want to try to create a reproducer. It will help me if I get some directions as I'm not experiences with sphinx at all.

cc @pat

Reply:
> Any idea how to get these properties total, total_found, etc. with plain SQL commands?

`SHOW META` after `SELECT`, e.g.:

```sql
mysql> drop table if exists t; create table t(f text); insert into t values(1, 'abc'),(2, 'abc'); select * from t where match('abc'); show meta;
--------------
drop table if exists t
--------------

Query OK, 0 rows affected (0.01 sec)

--------------
create table t(f text)
--------------

Query OK, 0 rows affected (0.00 sec)

--------------
insert into t values(1, 'abc'),(2, 'abc')
--------------

Query OK, 2 rows affected (0.00 sec)

--------------
select * from t where match('abc')
--------------

+------+------+
| id   | f    |
+------+------+
|    1 | abc  |
|    2 | abc  |
+------+------+
2 rows in set (0.01 sec)
--- 2 out of 2 results in 5ms ---

--------------
show meta
--------------

+----------------+-------+
| Variable_name  | Value |
+----------------+-------+
| total          | 2     |
| total_found    | 2     |
| total_relation | eq    |
| time           | 0.005 |
| keyword[0]     | abc   |
| docs[0]        | 2     |
| hits[0]        | 2     |
+----------------+-------+
7 rows in set (0.00 sec)
```

(Manticore Search v6)

Reply:
Thank you. I updated the upstream issue with a kind of a reproducer. btw it is not correct to use `total_found` instead of `total` according to sphinx docs, so this must be on manticore to fix:
https://sphinxsearch.com/blog/2014/03/28/basics-of-paginating-results/